GravatarApp/CommonViews/AnimatedHeaderScrollView/BouncyImageBackgroundHeaderView.swift

```diff
@@ -2,7 +2,7 @@ import GravatarUI
 import SwiftUI
 
 struct BouncyImageBackgroundHeaderView<Content>: View where Content: View {
-    let topSafeArea: CGFloat
+    let topPadding: CGFloat
```

Comment:
Bouncy will now receive the complete top padding, independent of if it's top safe are or something else.

This makes it more customizable, like incrementing the view height in special circumstances (like adding space for the refresh control while loading)

Is up to its parent to calculate correctly Bouncy's top padding acording to existence or not of top safe areas.